### PR TITLE
chore: migrate cypress to ts

### LINKED ISSUE #2: Migrate Cypress folder files to TS
Migrate all files from cypress folder to TS

### PR DESCRIPTION
**Pull request by YPAzevedo**
Fixes #2

### CODE REVIEW

**Comment by YPAzevedo** [comment]
Not really sure about the output and types declaration output. Let me know if this needs any changes.

File: cypress/integration/example-multi-grid-basic.spec.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/// <reference types="Cypress" />
+type GridUI = string | null | undefined
```
Comment (rbusquet) [inline cypress/integration/example-multi-grid-basic.spec.ts]:
```suggestion
type GridUID = string | null | undefined
```

Comment (YPAzevedo) [reply cypress/integration/example-multi-grid-basic.spec.ts]: Typo, thanks 🙏

**Review by nobrayner — APPROVED** [review]
Looks good to me! If we find anything wrong, we can just fix it as we go

:100: effort!!

**Comment by JacobMGEvans** [comment]
You even threw in some Cypress Best Practices!!! Nice work

**Review by JacobMGEvans — COMMENTED** [review]
Crushed it, great work

**Comment by YPAzevedo** [comment]
> You even threw in some Cypress Best Practices!!! Nice work

I did? lol. i guess yay me! 😂  What are the best practices? i just went with what made more sense to me

**Pull request merged**